test/test_helper.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
 require 'simplecov'
-SimpleCov.start 'rails'
+SimpleCov.start 'rails' do
+  add_filter 'app/models/schema.rb' # don't test the schema! https://github.com/rmosolgo/graphql-ruby/blob/master/guides/schema/testing.md#dont-test-the-schema
+  add_filter 'app/serializers' # these just contain the description of the attributes of the API responses, no code (https://github.com/Netflix/fast_jsonapi)
+  add_filter 'lib/graphql_collector.rb' # empty class that has to be there for prometheus integration
```

Comment:
We're using the codecov formatter for simplecov for codecov.io. This is the way to ignore files with plain old simplecov. To see accurate coverage locally (if you don't have the CODECOV_TOKEN), simplecov uses the HTMLFormatter, and so we have to use the simplecov filters to ignore files rather than codecov's config. This shouldn't affect the codecov formatter - it should continue to ignore these files like it used to.